help/actions/_posts/2015-02-09-export.md

```diff
@@ -74,27 +74,35 @@ The same format as above but encoded for use as a rich text document (rtf) which
+There are 16 author formats in the custom format. 
+
+Author formats for **last name** only
+
+| **and**                                                                                                                        	| **format**                                                       	| **specifier** | **abbreviated**       | **example**                                                                                                                    	| **example abbreviated %3.2**                                                            	|
+|--------------------------------------------------------------------------------------------------------------------------------	|------------------------------------------------------------------	|-----------	|----------------------	|--------------------------------------------------------------------------------------------------------------------------------	|-----------------------------------------------------------------------------------------	|
+| text                                                                                                                           	| first author only if<br>number of authors > 2<br>otherwise both authors 	| **%H**     |                      	| -Henneken<br>-Kurtz and Accomazzi                                                                                                  	|  -Henneken Muench<br>-Kurtz and Accomazzi                                                    	|
+| symbol                                                                                                                         	| first author only if<br>number of authors > 2<br>otherwise both authors 	| **%h**     |                      	|  -Henneken<br>-Kurtz & Accomazzi                                                                                                    	|  -Henneken Muench<br>-Kurtz & Accomazzi                                                             	|
+| text                                                                                                                           	| lastname                                                         	| **%M**     | , et al.             	|  -Henneken, Muench, Holm Nielsen, Blanco-Cuaresma, and Accomazzi<br>-Kurtz and Accomazzi                                            	|  -Henneken, Muench et al.-<br>Kurtz and Accomazzi                                            	|
+| symbol                                                                                                                         	| lastname                                                         	| **%m**     | , et al.             	|  -Henneken, Muench, Holm Nielsen, Blanco-Cuaresma, & Accomazzi<br>-Kurtz & Accomazzi                                                	|  -Henneken, Muench et al.<br>-Kurtz & Accomazzi                                              	|
```

Comment:
* Missing comma in `, et al.` in the "example abbreviated" example?

Reply:
oops, thank you. actually the issue is with `,` in the description. thankfully the examples are correct and do not need to fix the code. checked on the classic side.